portfolio.html

```diff
@@ -58,5 +58,176 @@ <h1 class="headline-heading">Hey there. I’m Norbi<br><span>I’m a software de
 
     </section>
 
+    <section class="work">
+        <h1 class="recent-works">My Recent Works</h1>
+        <img src="./images/mobile_version/Indicator/Indicator.png" class="indicator_horizontal" alt="indicator icon">
+
+        <div class="main-content-cards">
```

Comment:
- [ ] Kindly avoid using static widths in your design currently, all your cards are aligned to the left and they should be at the center please fix this
- [ ] please fix the first li; ruby on rails for each card, all li's should have the same width 👍🏿 


https://user-images.githubusercontent.com/69638013/185760961-5c8c82f8-eebb-40be-8d4f-5d411997345c.mp4